pkg/cmd/server/origin/auth.go

```diff
@@ -355,8 +357,11 @@ func (c *AuthConfig) getAuthenticationHandler(mux cmdutil.Mux, errorHandler hand
 				}
 				passwordSuccessHandler := handlers.AuthenticationSuccessHandlers{c.SessionAuth, redirectSuccessHandler{}}
 
+				loginPath := path.Join(OpenShiftLoginPrefix, identityProvider.Name)
```

Comment:
I don't see the advantage of that particular separation. I think that's probably over-engineering it. I'd say it's probably acceptable for us to just use the `OpenShiftLoginPrefix` for a single-provider configuration, but for _any_ multiple-provider scenario, we should always be specific. (Following the principle of least surprise)